docs/resources/website.md

```diff
@@ -189,3 +179,12 @@ Required:
 Read-Only:
 
 - `snippet` (String) The Website RUM monitoring code snippet (provided by the server).
+
+
+<a id="nestedatt--monitoring--options"></a>
+### Nested Schema for `monitoring.options`
+
+Required:
+
+- `is_availability_active` (Boolean) Is availability monitoring active?
+- `is_rum_active` (Boolean) Is RUM monitoring active?
```

Comment:
Shall we mark both of these as deprecated as well? I know the `options` parameter above is marked as deprecated, but maybe it is not entirely obvious.